avocado/utils/cpu.py

```diff
@@ -97,28 +97,20 @@ def get_cpu_arch():
     """
     Work out which CPU architecture we're running on
     """
+    cpu_table = [('^cpu.*(RS64|POWER3|Broadband Engine)', 'power'),
+                 ('^cpu.*POWER4', 'power4'),
+                 ('^cpu.*POWER5', 'power5'),
+                 ('^cpu.*POWER6', 'power6'),
+                 ('^cpu.*POWER7', 'power7'),
+                 ('^cpu.*POWER8', 'power8'),
+                 ('^cpu.*PPC970', 'power970'),
```

Comment:
@ldoktor there were no hurt *feelings* at all!

As I said on the comment, I fear the contribution from the community may be harmed, sometimes unintentionally.  Actually, even looking back at how this commit of mine was generated, I feel that I may have stepped into @sathnaga work... So I even apologize for that.

But, in fact, my comment was based on my understanding of *your understanding* of the 1st patch.  Now I know you missed some of its point, so even my comment wasn't 100% applicable here.

Anyway, thanks for the review!